release-notes/6.0/preview/api-diff/preview7/WindowsDesktop/6.0-preview7.md

```diff
@@ -0,0 +1,30 @@
+# API Difference 5.0.0 vs 6.0.0-preview.7.21378.9
+
+API listing follows standard diff formatting. Lines preceded by a '+' are
+additions and a '-' indicates removal.
+
+* [Microsoft.Win32](6.0-preview7_Microsoft.Win32.md)
+* [Microsoft.Win32.SafeHandles](6.0-preview7_Microsoft.Win32.SafeHandles.md)
+* [System.CodeDom.Compiler](6.0-preview7_System.CodeDom.Compiler.md)
+* [System.Configuration](6.0-preview7_System.Configuration.md)
+* [System.Diagnostics.Eventing.Reader](6.0-preview7_System.Diagnostics.Eventing.Reader.md)
+* [System.DirectoryServices.ActiveDirectory](6.0-preview7_System.DirectoryServices.ActiveDirectory.md)
```

Comment:
Arent these part of the normal .NET ref pack?

Reply:
These are all part of the windows desktop ref pack as well